### ISSUE TITLE
Error when building

### ISSUE THREAD

**Issue by tyeeman** [mattn/jedie#16]
I get this error when I build one of my .md files - 

```
jedie build
2019/10/23 22:09:49 jedie/_posts/2018-01-12-lite.md: [Error (where: parser) in <string> | Line 360 Col 32 near '.'] Expected either a number, string, keyword or identifier.
```
line 360 is this - 
```
below) like this: `<a href="/{{.Title | slug}}.html">A Post</a>`.
```
Col 32 is the dot before "Title". Any thing come to mind??

**Comment by mattn** [comment]
Please try `post.Title`

**Comment by tyeeman** [comment]
I don't think that works because this line is an instruction that tells how to get info using a go command from another application, not Jedie. Here is the full line - 
```
To get a post's page URL in a template use the `slug` function (described
below) like this: `<a href="/{{.Title | slug}}.html">A Post</a>`.
```
Can you see now? You can see the back ticks (`) which I think is saying this is code and copy it as written.

**Comment by mattn** [comment]
It is [pongo2](https://github.com/flosch/pongo2)'s behavior.

**Comment by tyeeman** [comment]
Is that a bug then, and I should mention it on their repo?

**Comment by mattn** [comment]
I'm not sure but probably you can use:

```
{% comment %}`<a href="/{{.Title | slug}}.html">A Post</a>`{% endcomment %}
```

**Comment by tyeeman** [comment]
OK, adding the comment tag allowed the build but left nothing in that space. 

Also during the build, it initially did not build with my files because I'm on Windows and use CRLF instead of LF only. I changed my files to LF only and the build worked.

**Comment by mattn** [comment]
>OK, adding the comment tag allowed the build but left nothing in that space.

OMG, Hmm, How about to escape `{{` to `%7b%7b` ?

**Comment by tyeeman** [comment]
I tried the backslashes but no difference.

**Comment by mattn** [comment]
Hmm, this is a limit of pongo2. If you want to solve this issue, please file new issue on pongo2.
